src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -392,6 +393,24 @@ function RequestedAmount({
             >
               <Checkbox checked={stable} onChange={onIsStableChange} />
               <span>Stable amount ({stableToken.symbol})</span>
+              {stable && (
```

Comment:
I wonder if we should always show the Help tip.

Reply:
Let´s do it